modules/logging-loki-log-access.adoc

```diff
@@ -74,16 +74,36 @@ subjects:
 // tag::CustomAdmin[]
 == Custom admin group access
 
-If you have a large deployment with a number of users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. Admin users have access to all application logs in all namespaces, if they also get assigned the `cluster-logging-application-view` role.
+// tag::LokiMode[]
+If you have a large deployment with several users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered administrators. 
```

Comment:
Since this is the first mention of CR in the section:
```suggestion
If you have a large deployment with several users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` custom resource (CR) are considered administrators. 
```

Reply:
Addressed in comment below.